### PR TITLE
New recipe for DIALIGN-TX

### PR DESCRIPTION
**Pull request by jhchung**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by jhchung** [comment]
Hi @bioconda/core, I'm a new contributor and would like to contribute a new recipe for DIALIGN-TX. It's a program for segment-based multiple sequence alignment. 

Thanks.

File: recipes/dialign-tx/build.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -euo pipefail
+export CPPFLAGS="-I$PREFIX/include"
+export LDFLAGS="-L$PREFIX/lib"
+sed -i -e 's/-march=i686/-march=native/' ./source/Makefile
```
Comment (bgruening) [inline recipes/dialign-tx/build.sh]: For OSX you probably need `sed -i.bak`

File: recipes/dialign-tx/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  url: http://dialign-tx.gobics.de/DIALIGN-TX_1.0.2.tar.gz 
+  md5: 8ccfb1d91136157324d1e513f184ca29
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - gcc  # [not osx]
+    - zlib
```
Comment (bgruening) [inline recipes/dialign-tx/meta.yaml]: please add `{{CONDA_ZLIB}}*` to pin zlib, also in the run section :)

File: recipes/dialign-tx/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+  run:
+    - libgcc  # [not osx]
+    - zlib
+
+test:
+    script: run_test.sh
```
Comment (bgruening) [inline recipes/dialign-tx/meta.yaml]: You can inlude `dialign-tx -h | grep 'Usage' > /dev/null` here directly by using the command tag.

File: recipes/dialign-tx/meta.yaml (hunk 2 above)
Comment (jhchung) [reply recipes/dialign-tx/meta.yaml]: I actually think i need zlib, leftover from a skeleton meta file. Still builds fine on my local system without it.

**Comment by jhchung** [comment]
@bgruening Made the requested changes. Thanks for the comments.

**Comment by bgruening** [comment]
Have you used `simulate_travis` locally? If you haven't it can be that you simply used the zlib from your host.

**Comment by jhchung** [comment]
yes, I ran `./simulate-travis.py --disable-docker` and it passed all the tests. I am disabling docker because I can't get it installed on my system right now.

**Comment by druvus** [comment]
@jhchung I have successfully added dialign-tx in PR #6423 so I will close this now.

**Pull request closed without merging**